src/main/java/com/powsybl/openloadflow/network/impl/LfBatteryImpl.java

```diff
@@ -93,6 +93,20 @@ public double getMaxP() {
         return getBattery().getMaxP() / PerUnit.SB;
     }
 
+    @Override
+    public double getMinTargetP() {
+        Battery battery = getBattery();
```

Comment:
Why do you prefer to do everything here rather than do it at creation through 2 attributes? Same remark for generator impl.